routes/cube/api.js

```diff
@@ -431,6 +431,39 @@ router.get(
   }),
 );
 
+router.get(
```

Comment:
Prefer post for API endpoints

Reply:
Im happy to switch to POST, but I'm curious why CubeCobra prefers POST to GET in this situation

Reply:
Also with the switch to POST, would you prefer lastKey use the request body instead of a search param?